recipe/meta.yaml

```diff
@@ -17,9 +17,9 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
-  skip: true  # [not linux64 and cuda_compiler_version != "None"]
-  skip: true  # [linux64 and cuda_compiler_version == "10.2"]
+  number: 2
+  skip: true  # [not linux and cuda_compiler_version != "None"]
+  skip: true  # [linux and cuda_compiler_version == "10.2"]
```

Comment:
As conda-forge requires CUDA 11.2+ ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1708 ), think we can drop this CUDA 10.2 `skip` altogether

```suggestion
```